test/LoadTest/README.md

```diff
@@ -0,0 +1,157 @@
+## JMeter scripts
+
+This is the Jmeter scripts repository for the Orion Context Broker, used for Performance tests.
+
+#### Pre-conditions:
+
+* "Jmeter" app exists in Launcher VM 
+* "ServerAgent" app exist in Launcher VM, in ContextBroker VM and Nginx VM (only in cluster case) 
+* have a account in Loadosophia - (http://loadosophia.org)
+* "nginx" app exists in Balancer VM (only in cluster case)
+* Verify nginx configuration for each scenario (only in cluster case)
+* Verify contextBroker MD5 in each Node
+* Verify mongoDB version in each Node	
+	
+#### Steps:
+
+* Launch "ServerAgent" in ContextBroker VM in Balancer and each Node VM 
+```
+nohup sh startAgent.sh --udp-port 0 --tcp-port 3450 > monitor.log &
+```
+* Launch jmeter script "orionPerfTest_cluster_v1.2.jmx" in Launcher VM
+```
+./jmeter.sh -n -t <script>.jmx -J<property>=value > /<path>/jmeter_report__`date +%Y%m%d%H`.log
+```
+* Upload in Loadosophia Loadosophia_xxxxxxxxxxxxxxxxxxxxx.jtl.gz and perfmon_xxxxxxxxxxxxxxxxxxxx.jtl.gz (where "xxxxxxxxxxxxxxxxxxx" is a hash value).
+
+Comments:
+    /tmp/error_xxxxxxxxxxxxxxxxxxx.html is created, because does not have access at loadosophia, the token is wrong intentionally 
+is made to not constantly access and penalizes the test times. We only store datas manually when finished test. So "xxxxxxxxxxxxxxxxxxx" is a hash value.
+
+#### Scripts:
+1. **orionPerformanceTest.jmx**   (used by Max Performance, Mongo Impact, Scale UP, Scale OUT). It can used for one standalone VM or a balanced cluster of VM with 4 nodes maximum.
```

Comment:
Looking to directory contents, the file seems to be named orionPerformanceTest_v1.0.jmx. In that case, README.md should use the right name.